### DISCUSSION TITLE
Is it possible to change titlebar color (flatpak release)

### DISCUSSION THREAD

**Discussion by cmaspi** [ahrm/sioyek#1058] [Q&A]
Is it currently possible to change the color of the titlebar?

At present it is white and it looks really annoying in dark mode.

I saw a PR that added the feature for mac, does it work with linux?

**Comment by ahrm** [comment]
It doesn't work with linux. I am not sure if there is a standard way to do that in linux, probably every window manager has its own way of doing this.

**Comment by cmaspi** [comment]
Update: The beta release as of writing this post allows for a dark titlebar.
